lib/ExpensiMark.js

```diff
@@ -250,6 +250,29 @@ export default class ExpensiMark {
                 }
             },
         ];
+
+        /**
+         * The list of rules to covert the HTML to text.
+         * Order of rules is important
+         * @type {Object[]}
+         */
+        this.htmlToTextRules = [
+            {
+                name: 'breakline',
+                regex: /((<br[^>]*>)+)/gi,
+                replacement: ' ',
+            },
+            {
+                name: 'blockquote',
+                regex: /<\/blockquote>/gm,
+                replacement: ' ',
```

Comment:
I'm surprised the tests with blockquotes work, considering that the

```
const html = '<blockquote>Confusing stuff</blockquote>Tell me about it';
const text = 'Confusing stuff Tell me about it';
```

Why isn't there a space before the word "Confusing" here?

Reply:
Also, it seems we're only capturing the closing tag. I must be wrong about that though because the tests are passing.

Reply:
> Why isn't there a space before the word "Confusing" here?

> Also, it seems we're only capturing the closing tag. I must be wrong about that though because the tests are passing.


Here we loop the 3 rules 
1. replacing breakline with space `/((<br[^>]*>)+)/gi`
2. replacing blockquote with space `/<\/blockquote>/gm`. **This the solution for the issue**
3. strip all the tags left behind with an empty string `/(<([^>]+)>)/gi`

```js
        this.htmlToTextRules.forEach((rule) => {
            replacedText = replacedText.replace(rule.regex, rule.replacement);
        });
```

@parasharrajat @Luke9389